.github/workflows/cancel.yml

```diff
@@ -0,0 +1,13 @@
+name: Cancel
+on:
+  workflow_run:
+    workflows: ["Ensure PR"]
+    types:
+      - requested
```

Comment:
```suggestion
on:
  pull_request:
```

Reply:
Nah, it won't run against those pull requests from forks https://github.com/styfle/cancel-workflow-action#advanced-pull-requests-from-forks. E.g, I'm submitting this pull request from my forked repository.

Reply:
[Try this](https://github.com/webpack-contrib/css-loader/blob/master/.github/workflows/cancel.yml#L2), we have similar in webpack-contrib and it runs on PR form forks.

Reply:
Thanks. Any reason to avoid `workflow_run`? Seems fine to me according to the documentation.

Reply:
Maybe we can only test it after getting it merged, not in this one.

Reply:
I'm good with it if it works 👍